make_queue/views/reservation/reservation.py

```diff
@@ -182,7 +202,8 @@ def dispatch(self, request, *args, **kwargs):
         :param request: The HTTP request
         """
         # User must be able to change the given reservation
-        if not kwargs["reservation"].can_change(request.user) and not kwargs["reservation"].can_change_end_time(
+        if not kwargs["reservation"].can_change(
+                request.user) and not kwargs["reservation"].can_change_end_time(
                 request.user):
```

Comment:
The way this line is separated makes it a bit hard to read. I would suggest changing it to this instead:
```python
if not kwargs["reservation"].can_change(request.user)
   and not kwargs["reservation"].can_change_end_time(request.user):
```
or
```python
if not (kwargs["reservation"].can_change(request.user)
        and kwargs["reservation"].can_change_end_time(request.user)):
```